src/fragments/start/getting-started/reactnative/integrate.mdx

```diff
@@ -0,0 +1,411 @@
+
+```js
+import React from 'react';
+import { StatusBar, StyleSheet, View } from 'react-native';
+import Amplify from 'aws-amplify';
+import Home from './src/Home';
+
+import awsconfig from './src/aws-exports';
+
+Amplify.configure(awsconfig);
+
+export default function App() {
+  return (
+    <View style={styles.container}>
+      <StatusBar />
+      <Home />
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: '#fff',
+    flex: 1,
+  },
+});
+```
+
+#### Home.js
+Next, we will create a Home component for our app which will implement most of the CRUD functionality for our app. Create a new file `Home.js` at location `src/Home.js`. Here, we have total four components defined :
+
+1. **Header**: Simple Header component with Title.
+2. **AddModal**: Used for displaying a Modal when a new Todo needs to be added.
+3. **TodoList**: Used for displaying the list of Todos.
+4. **Home**: Default component that wraps all the above component and a Button for adding a new Todo item.
+
+Copy the below boilerplate UI code into the newly create file.
+
+```js
+import React, { useState, useEffect } from 'react';
+import {
+  FlatList,
+  Modal,
+  Pressable,
+  StyleSheet,
+  Text,
+  TextInput,
+  View,
+} from 'react-native';
+import { DataStore } from 'aws-amplify';
+import { Todo } from './models';
+
+const Header = () => (
+  <View style={styles.headerContainer}>
+    <Text style={styles.headerTitle}>My Todo List</Text>
+  </View>
+);
+
+const AddTodoModal = ({ modalVisible, setModalVisible }) => {
+  const [name, setName] = useState('');
+  const [description, setDescription] = useState('');
+
+  async function addTodo() {
+  }
+
+  function closeModal() {
+    setModalVisible(false);
+  }
+
+  return (
+    <Modal
+      animationType="fade"
+      onRequestClose={closeModal}
+      transparent
+      visible={modalVisible}
+    >
+      <View style={styles.modalContainer}>
+        <View style={styles.modalInnerContainer}>
+          <Pressable onPress={closeModal} style={styles.modalDismissButton}>
+            <Text style={styles.modalDismissText}>X</Text>
+          </Pressable>
+          <TextInput
+            onChangeText={setName}
+            placeholder="Name"
+            style={styles.modalInput}
+          />
+          <TextInput
+            onChangeText={setDescription}
+            placeholder="Description"
+            style={styles.modalInput}
+          />
+          <Pressable onPress={addTodo} style={styles.buttonContainer}>
+            <Text style={styles.buttonText}>Save Todo</Text>
+          </Pressable>
+        </View>
+      </View>
+    </Modal>
+  );
+};
+
+const TodoList = () => {
+  const [todos, setTodos] = useState([]);
+
+  useEffect(() => {
+
+  }, []);
+
+  async function deleteTodo(todo) {
+  }
+
+  async function setComplete(updateValue, todo) {
+  }
+
+  const TodoItem = ({ item }) => (
```

Comment:
Nit: I think it is idiomatic to just call this a `renderItem` function or even to place it inline of the FlatList rather than treating this as a component being defined within another component. You can see in RN's docs this is how they have it in their sample snippets as well.
```suggestion
  const renderItem = ({ item }) => (
```

Reply:
Updated the name of the function to renderItem.